webapp/src/main/webapp/i18n/vivo_all.properties

```diff
@@ -810,3 +810,55 @@ role_in_presentation_capitalized=Role in Presentation
 advisee_capitalized_first_name=First Name
 advisee_capitalized_lastname=Last Name
 
+# Messages for creating and linking resources (publications)
```

Comment:
These should also be included in https://github.com/vivo-project/VIVO-languages, no?

Reply:
Created a PR for VIVO-languages using the English i18n strings: https://github.com/vivo-project/VIVO-languages/pull/20